vizro-core/src/vizro/models/_components/table.py

```diff
@@ -31,10 +31,13 @@ class Table(VizroBaseModel):
     title: Optional[str] = Field(None, description="Title of the table")
     actions: List[Action] = []
 
+    _underlying_table_id: str = PrivateAttr()
+
```

Comment:
Once we support AgGrid and others, we'll need to handle the `"active_cell"` argument in the following line:

`set_actions = _action_validator_factory("active_cell")`